restic/internal/cat_test.py

```diff
@@ -0,0 +1,261 @@
+import unittest
+from unittest import mock
+
+import restic
+from restic.internal import cat
+
+
+class CatPackTest(unittest.TestCase):
+
+    @mock.patch.object(cat.command_executor, 'execute')
+    def test_cat_pack(self, mock_execute):
+        mock_execute.return_value = '{}'
+
+        restic.cat.pack(
+            'b5bb9d8014a0f9b1d61e21e796d78dccdf1352f23cd32812f4850b878ae4944c')
+
+        mock_execute.assert_called_with([
+            'restic', '--json', 'cat', 'pack',
+            'b5bb9d8014a0f9b1d61e21e796d78dccdf1352f23cd32812f4850b878ae4944c'
+        ])
+
+
+class CatBlobTest(unittest.TestCase):
+
+    @mock.patch.object(cat.command_executor, 'execute')
+    def test_cat_blob(self, mock_execute):
+        mock_execute.return_value = """{"nodes":[
+{"name":"bar", "type":"file", "mode":420, "mtime":"2022-12-09T08:36:42.052086879+01:00",
+ "atime":"2022-12-09T08:36:42.052086879+01:00","ctime":"2022-12-09T08:36:42.052086879+01:00",
+ "uid":501,"gid":0,"user":"user1","group":"wheel","inode":10367333,"device_id":16777232,
+ "links":1,"content":[]}]}""".lstrip()
```

Comment:
We shouldn't need the `lstrip` since there's no whitespace at the beginning of the string.

(ditto throughout)